airship_keyboard/Gain gain1
airship_keyboard/Gain gain2
airship_keyboard/Gain gain3
#Gain gainN
---
bool success   # indicate successful run of triggered service

